app/models/git_repository.rb

```diff
@@ -155,6 +183,36 @@ def handling_worktree_errors
     raise MiqException::Error, err.message
   end
 
+  def git_lock_filename
+    @git_lock_filename ||= LOCKFILE_DIR.join(id.to_s)
```

Comment:
Do we need a method + instance variable when there is only one usage of this?

Reply:
No.  Did it for readability mostly.